datacapture/src/main/java/com/google/android/fhir/datacapture/extensions/MoreQuestionnaireItemAnswerOptionComponents.kt

```diff
@@ -42,6 +44,10 @@ internal val Questionnaire.QuestionnaireItemAnswerOptionComponent.optionExclusiv
     return false
   }
 
+/** Get the answer options values with `initialSelected` set to true */
+internal val List<QuestionnaireItemAnswerOptionComponent>.initial: List<Type>
```

Comment:
can you rename this to initialSelected?

```suggestion
internal val List<QuestionnaireItemAnswerOptionComponent>.initialSelected: List<Type>
```